src/main/java/uk/gov/justice/hmpps/offenderevents/services/EventRetrievalService.java

```diff
@@ -122,7 +124,7 @@ public void runTestPolls(final LocalDateTime now) {
                 () -> {
                     // First run
                     final LocalDateTime startTimeA = endTimeA.minus(1, MINUTES);
-                    final var eventsA = externalApiService.getEvents(startTimeA, endTimeA);
+                    final var eventsA = externalApiService.getTestEvents(startTimeA, endTimeA, useEnq);
                     final var countA = eventsA.size();
```

Comment:
I didnt bother with custom events as I can extract the counts from the log message (and do this for the original poller too for comparison)